docs/logging.md

```diff
@@ -235,20 +221,28 @@ to resolve a program. When the interpreter starts, the RESL channel logs the lis
 of strategies used to locate a RPG/Java program.
 
 ```
-15:09:46.910                  RESL resource: /
-15:09:46.910                  RESL directory: .
-15:09:46.960 TEST_06       80 RESL CALL "CALCFIB"
-+-----------+-------------+--+---+---------- resolution -----------+
+15:47:40.824 RESL	T10_A60_P02	8	CALL	"MULANGTC10"
++------------ header -------------+---- resolution ----+
+```
+
+## Analytics Channel ANALYTICS
+The analytics channel provides processed collected during the whole execution cycle. Information yield by this channel are derived by a LoggingContext where logging metadata are stored.
+
+```
+10:24:41.060 ANALYTICS    MUTE10_10    STMT TIME    DO    451457us    33%    1    avg. 451457us
```

Comment:
I don't understand why the sum in percent is 62%.
It misses the time spent by program